config/environments/production.rb

```diff
@@ -21,6 +21,8 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
+  config.cache_store = :redis_cache_store, { url: ENV.fetch('REDIS_CACHE_URL') }
```

Comment:
Not sure how this is here? It's already in main

Reply:
Ah, quite right! Apologies, branching off the wrong base branch. Will rework this PR for just the dev config.